library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_backward_data.hpp

```diff
@@ -100,6 +100,91 @@ void add_device_grouped_conv2d_bwd_data_xdl_nhwgc_gkyxc_nhwgk_bf16_instances(
                                                                   PassThrough,
                                                                   PassThrough>>>& instances);
 
+// conv3d backward data
+void add_device_grouped_conv3d_bwd_data_xdl_gndhwc_gkzyxc_gndhwk_f16_instances(
```

Comment:
The name of this function is not consistent with the Kernel input data layout order. There is: in, wei, out and should be out, wei, in

Reply:
Done.